metadata.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "bbriggs-rocketchat",
+  "version": "0.1.0",
+  "author": "bbriggs",
```

Comment:
probably should be `voxpupuli`.  @bastelfreak - Do we have a policy?  For adopted modules, should we actually also credit the original author here?  Does the forge use this parameter for anything??